src/redux/cart/cart.operations.js

```diff
@@ -107,6 +107,13 @@ const getCartByUserId = async (userId) => {
   `;
   const result = await getItems(getCartByUserIdQuery, { id: userId });
 
+  if (
+    result?.data?.getCartByUserId?.message &&
+    result?.data?.getCartByUserId?.message !== 'CART_IS_NOT_FOUND'
```

Comment:
remove `result?.data?.getCartByUserId?.message !== 'CART_IS_NOT_FOUND'` this useless check